packages/onboard-react/src/components/WalletDialog/WalletDialog.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC, useCallback, useMemo, useState } from "react";
+import {
+  Dialog,
+  VerticalTabs,
+  VerticalTabsItem,
+  Button,
+  ButtonStyle,
+  Heading,
+  DialogSize,
+} from "@sovryn/ui";
+import { FilterType, WalletList } from "../WalletList/WalletList";
+import { connectWallet$ } from "@sovryn/onboard-core/dist/streams";
```

Comment:
Copy prettier config from `sovryn-dapp` to this repo :)

Reply:
I think config should another task - it's out of scope to the story and this PR would get hundreds of unrelated files changed which you would need to review :)

Reply:
agreed, will create another task for that update (it's not high priority)